modules/dataframe/include/inviwo/dataframe/datastructures/column.h

```diff
@@ -78,7 +78,8 @@ std::basic_ostream<Elem, Traits>& operator<<(std::basic_ostream<Elem, Traits>& s
 }
 
 /**
- * @brief pure interface for representing a data column, i.e. a Buffer with a name
+ * @brief Pure interface for representing a data column, i.e. a Buffer with a name
```

Comment:
Don't think buffer should be mentioned. that seems like an implementation detail. name should be header, and might also add unit and range.